R/annotation-custom.r

```diff
@@ -73,7 +73,12 @@ GeomCustomAnn <- ggproto("GeomCustomAnn", Geom,
     if (!inherits(coord, "CoordCartesian")) {
       cli::cli_abort("{.fn annotation_custom} only works with {.fn coord_cartesian}")
     }
-    corners <- new_data_frame(list(x = c(xmin, xmax), y = c(ymin, ymax)), n = 2)
+    corners <- data_frame(
+      x = c(xmin, xmax),
+      y = c(ymin, ymax),
+      .size = 2,
+      .name_repair = "minimal"
```

Comment:
You shouldn't need `.name_repair` here, right? Because none of the names are user supplied?

Reply:
``` r
bench::mark(vctrs::data_frame(x=1,y=2),vctrs::data_frame(x=1,y=2,.name_repair = "minimal"))
#> # A tibble: 2 × 6
#>   expression                                                     min   median
#>   <bch:expr>                                                <bch:tm> <bch:tm>
#> 1 vctrs::data_frame(x = 1, y = 2)                            24.39µs  31.61µs
#> 2 vctrs::data_frame(x = 1, y = 2, .name_repair = "minimal")   5.15µs   5.89µs
#> # … with 3 more variables: `itr/sec` <dbl>, mem_alloc <bch:byt>, `gc/sec` <dbl>
```

<sup>Created on 2022-06-15 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>